tests/Fixtures/ConvertConfig/full_configuration.yaml

```diff
@@ -0,0 +1,36 @@
+imports:
```

Comment:
This is the most complete configuration which is used to test that all config options work together correctly